modules/host.go

```diff
@@ -5,21 +5,25 @@ import (
 )
 
 const (
-	AcceptTermsResponse = "accept"
-	HostDir             = "host"
+	AcceptResponse = "accept"
+	HostDir        = "host"
 )
 
-// ContractTerms are the parameters agreed upon by a client and a host when
-// forming a FileContract.
-type ContractTerms struct {
-	FileSize           uint64                // How large the file is.
-	Duration           types.BlockHeight     // How long the file is to be stored.
-	DurationStart      types.BlockHeight     // The block height that the storing starts (typically required to start immediately, unless it's a chained contract).
-	WindowSize         types.BlockHeight     // How long the host has to submit a proof of storage.
-	Price              types.Currency        // Client contribution towards payout each window
-	Collateral         types.Currency        // Host contribution towards payout each window
-	ValidProofOutputs  []types.SiacoinOutput // Where money goes if the storage proof is successful.
-	MissedProofOutputs []types.SiacoinOutput // Where the money goes if the storage proof fails.
+// RPC identifiers
+var (
+	// Each identifier has a version number at the end, which will be
+	// incremented whenever the protocol changes.
+	RPCSettings = types.Specifier{'S', 'e', 't', 't', 'i', 'n', 'g', 's', 0}
```

Comment:
Blocks and transactions are both handled by the gateway, which hasn't been changed since v0.3.3.3. So they should continue to propagate normally, as long as the consensus set and transaction pool are compatible as well.